docs/lang/articles/performance_tuning/profiler.md

```diff
@@ -105,11 +105,8 @@ X64 Profiler(count)
 
 :::note
 
-- A kernel that is fully optimized by the compiler does not generate profiling records.
-- One kernel may generate multiple records of parallel for loops because they are divided into different tasks and assigned to separate devices.
-- `kernel_profiler` supports CPU and CUDA only. Ensure that you call `ti.sync()` before performance profiling if your program is running on GPU.
 - `jit_evaluator_xxx` can be ignored because it is automatically generated by the system.
-- Taichi recommends running performance profiling multiple times to observe the minimum or average execution time.
+- You are recommended running performance profiling multiple times to observe the minimum or average execution time.
```

Comment:
I edited this line before. 
Correct: "Taichi recommends doing" or "Taichi recommends that you do". 
Incorrect: You are recommended doing. 

At least it ought to be "you are recommended to do".

Reply:
I suggest you to download a plugin called GitLens from vs code marketplace to avoid duplicate edits.